pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -246,6 +246,20 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 			Message: ReasonRunning,
 		})
 	}
+	tr.Status.StartTime = build.Status.StartTime
+	if build.Status.Cluster != nil {
+		tr.Status.PodName = build.Status.Cluster.PodName
+	}
+	tr.Status.CompletionTime = build.Status.CompletionTime
+
+	tr.Status.Steps = []v1alpha1.StepState{}
+	for i := 0; i < len(build.Status.StepStates); i++ {
+		state := build.Status.StepStates[i]
+		tr.Status.Steps = append(tr.Status.Steps, v1alpha1.StepState{
+			ContainerState: *state.DeepCopy(),
+			// FIXME(vdemeester) logsURL
```

Comment:
if this is something you intend to fix in a further PR can you add a bit more detail about what needs to happen here?

if you plan to fix it in this PR just ignore me :D

Reply:
Ah right, I should add more context to this… So there is a `LogsURL` field in the `StepState` (previously `StepRun`), which was not populate (and still is not). That's why the FIXME here, to *not forget*… but really, to not forget we shoul create an issue, not a comment :stuck_out_tongue_closed_eyes: